catalog/ui/catalog-ui-search/src/main/webapp/component/metacard-interactions/metacard-interactions.view.js

```diff
@@ -146,7 +146,10 @@ define([
             }, '');
             let cqlString = "(" + combinedCqlString + ")";
             var newQuery = new Query.Model({
-                type: locations.length > 1 ? 'advanced' : 'basic'
+                type: locations.length > 1 ? 'advanced' : 'basic',
```

Comment:
This is the one case where I'm not sure if we should actually be using the user's defaults for queries instead.